polyfill/lib/ecmascript.mjs

```diff
@@ -269,12 +269,13 @@ export const ES = ObjectAssign({}, ES2022, {
       var excluded = some(excludedKeys, function (e) {
         return SameValue(e, nextKey) === true;
       });
+      if (excluded) return;
 
       var enumerable =
```

Comment:
Since we're changing this function anyways, should we replace var with let or const?

Reply:
Richard wanted to keep it the same as the original CopyDataProperties as much as possible.

Reply:
OK